qiskit/pulse/__init__.py

```diff
@@ -13,56 +13,77 @@
 # that they have been altered from the originals.
 
 """
-===============================
-OpenPulse (:mod:`qiskit.pulse`)
-===============================
-
 .. currentmodule:: qiskit.pulse
 
-Channels
-========
+===========================
+Pulse (:mod:`qiskit.pulse`)
+===========================
 
-.. autosummary::
-   :toctree: ../stubs/
+Qiskit-Pulse is a pulse-level quantum programming kit. This lower level of programming offers the
+user more control than programming with :py:class:`~qiskit.circuit.QuantumCircuit` s.
 
-   DriveChannel
-   MeasureChannel
-   AcquireChannel
-   ControlChannel
-   RegisterSlot
-   MemorySlot
+Extracting the greatest performance out from quantum hardware requires real-time pulse-level
+instructions. Pulse answers that need: it enables the quantum physicist *user* to specify the
+exact time dynamics of an experiment. It is especially powerful for error mitigation techniques.
+
+The input is given as arbitrary, time-ordered signals (see: :ref:`pulse-commands`) scheduled in
+parallel over multiple virtual hardware or simulator resources (see: :ref:`pulse-channels`). The
+system also allows the user to recover the time dynamics of the measured output.
+
+This is sufficient to allow the quantum physicist to explore and correct for noise in a quantum
+system.
 
-Commands
-========
+.. _pulse-commands:
+
+Commands (:mod:`~qiskit.pulse.commands.command`)
+================================================
 
 .. autosummary::
    :toctree: ../stubs/
 
-   Instruction
-   Acquire
-   AcquireInstruction
-   FrameChange
-   PersistentValue
    SamplePulse
-   Snapshot
-   Kernel
-   Discriminator
    Delay
-   ParametricPulse
-   ParametricInstruction
+   FrameChange
    Gaussian
    GaussianSquare
    Drag
    ConstantPulse
+   Acquire
+   Snapshot
+
+.. _pulse-channels:
+
+Channels (:mod:`~qiskit.pulse.channels`)
+========================================
+
+Pulse is meant to be agnostic to the underlying hardware implementation, while still allowing
+low-level control. Therefore, our signal channels are  *virtual* hardware channels. The backend
+which executes our programs is responsible for mapping these virtual channels to the proper
+physical channel within the quantum control hardware.
+
+Channels are characterized by their type and their index. See each of the below to learn more.
```

Comment:
I think something like 'channel type', 'channel class', or even 'channel-type class' (lol) will make this last sentence read better.

```suggestion
Channels are characterized by their type and their index. See each channel type below to learn more.
```